pkgs/applications/networking/kubo/default.nix

```diff
@@ -2,15 +2,15 @@
 
 buildGoModule rec {
   pname = "kubo";
-  version = "0.20.0"; # When updating, also check if the repo version changed and adjust repoVersion below
+  version = "0.21.0"; # When updating, also check if the repo version changed and adjust repoVersion below
   rev = "v${version}";
 
   passthru.repoVersion = "13"; # Also update kubo-migrator when changing the repo version
```

Comment:
```suggestion
  passthru.repoVersion = "14"; # Also update kubo-migrator when changing the repo version
```

Reply:
Indeed when launching a node with the old version it says it needs to update the repo. I didn't find anything in the changelog about this, how did you know this version should be bumped as well?

Also, should I upgrade the kubo-migrator in a separate commit?

Reply:
Strike my question about the migrator, I see that it already support the 13 to 14 migration.

Reply:
Yeah, it's not obvious in the changelog, all it says is "A config migration has been added to handle this automatically". You basically have to check manually. When a new version releases, I either already saw a new release of the migrator program or I execute `ipfs init` and then check the contents of `~/.ipfs/version`. In this case, I already added the migration in #240311.